### PR TITLE
Add github action to run linux/arm64 integration tests

### PR DESCRIPTION
**Pull request by ilya-zuyev**
This PR adds two new GitHub actions:
- run functional integration test set for new PRs
- run full integration test set for new PRs

### CODE REVIEW

**Comment by tstromberg** [comment]
Seems like there is something missing:

`/home/ubuntu/actions-runner/_work/_temp/081d8436-fe62-46e9-9036-a89f41f9f46b.sh: line 7: ./e2e-linux-arm64: No such file or directory`

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -649,6 +649,145 @@ jobs:
           if [ "$numPass" -eq 0 ];then echo "*** 0 Passed! ***";exit 2;fi
           if [ "$numPass" -lt 26 ];then echo "*** Failed to pass at least 26 ! ***";exit 2;fi
           if [ "$numPass" -eq 0 ];then echo "*** Passed! ***";exit 0;fi
+  functional_docker_ubuntu_arm64:
```
Comment (medyagh) [inline .github/workflows/pr.yml:652]: add same things to master.yml

and also ensure the arm64 binary is being copied from Build Step to the job
and ensure to use gopogh

**Review by medyagh — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (medyagh) [inline .github/workflows/pr.yml:652]: ensure it is added to both pr.yml and master.yml

**Review by medyagh — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -649,6 +649,145 @@ jobs:
...
+    env:
+      TIME_ELAPSED: time
+      JOB_NAME: "functional_docker_ubuntu"
+      GOPOGH_RESULT: ""
+      SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
+    steps:
+      - name: Install kubectl
+        shell: bash
+        run: |
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/arm64/kubectl
```
Comment (medyagh) [inline .github/workflows/pr.yml:664]: lets add installing gopogh here too.

**Review by medyagh — CHANGES_REQUESTED** [review]

**Comment by medyagh** [comment]
test arm64 says 
```
/home/ubuntu/actions-runner/_work/_temp/081d8436-fe62-46e9-9036-a89f41f9f46b.sh: line 7: ./e2e-linux-arm64: No such file or directory
```

I think the make cross did not build the arm64 or it was not copied over https://github.com/medyagh/minikube/blob/53c73e5c19af45508c27b433545668fa093aa5ab/.github/workflows/pr.yml#L29

File: .github/workflows/pr.yml (hunk 1 above)
Comment (ilya-zuyev) [reply .github/workflows/pr.yml:653]: done

File: .github/workflows/pr.yml (hunk 2 above)
Comment (ilya-zuyev) [reply .github/workflows/pr.yml:665]: done

**Review by medyagh — APPROVED** [review]
looks good to me, lets add another agent, (currently it showes Qeued)
and also lets add clean up cronjob in a follow up pr

https://github.com/kubernetes/minikube/issues/10179

**Review by medyagh — CHANGES_REQUESTED** [review]
lets add command to install jq or make or anything needed for the job to run

**Review by medyagh — APPROVED** [review]
thank you looks good !

**Pull request merged**